ui/summit-2024/src/pages/Home/components/TicketsSection.tsx

```diff
@@ -174,7 +180,7 @@ const TicketsSection = () => {
                     marginTop: "32px",
                   }}
                 >
-                  <CustomButton colorVariant="primary">
+                  <CustomButton colorVariant="primary" onClick={handleButtonClick}>
                     Get Tickets
```

Comment:
Get Tickets should be in the i18n en.json file I would guess but not relevant to this PR